charts/tenant-ui/values-development.yaml

```diff
@@ -21,7 +21,7 @@ tenant_ui:
     server: apps.smtp.gov.bc.ca
     port: 25
     senderAddress: DoNotReplyTractionDEV@gov.bc.ca
-    innkeeperInbox: jason.sherman@gmail.com
+    innkeeperInbox: "lucas.o'neil@gov.bc.ca,emiliano.sune@quartech.com"
```

Comment:
@esune let me know if you don't want to be on here for the dev env, or if we want other innkeeper contacts or anything

Reply:
I'm fine being there. This env will be re-configured soon anyway, and having a backup for approvals seems like a good idea.